### PR TITLE
Use a simple get/set accessor instead

### PR DESCRIPTION
**Pull request by bwoebi**
As we ended up disagreeing on a common global accessor, I propose simplifying the whole accessor to just a get()+set() method pair.

This ensures that the loop will be interoperable at all (otherwise we'd end up having different loops active or worse things).

Now, every library can choose its own accessors as desired.
This should resolve the current point of contention.

Is this something we all can agree on?
\cc @kelunik @trowski @clue @WyriHaximus @martinschroeder @jsor @cboden @davidwdan @mbonneau @rdlowrey (please leave a thumbs up or down)

After that, I guess we can relatively quickly go to tagging v1 then as I haven't seen much objection on the Driver interface itself at all.

### CODE REVIEW

File: src/Loop.php [hunk 1]
```diff
@@ -3,423 +3,37 @@
 namespace AsyncInterop;
 
 use AsyncInterop\Loop\Driver;
-use AsyncInterop\Loop\DriverFactory;
-use AsyncInterop\Loop\InvalidWatcherException;
-use AsyncInterop\Loop\UnsupportedFeatureException;
 
 /**
- * Accessor to allow global access to the event loop.
+ * Accessor to allow global access to the event loop
```
Comment (kelunik) [inline src/Loop.php:8]: Missing dot.

File: src/Loop.php [hunk 2]
```diff
@@ -3,423 +3,37 @@
...
-        }
-
-        return $driver;
-    }
-
-    /**
-     * Retrieve the event loop driver that is in scope.
+     * Retrieve the current event loop driver
      *
      * @return Driver
```
Comment (kelunik) [inline src/Loop.php]: Must be `@return Driver|null` then.

Comment (bwoebi) [reply src/Loop.php]: right, will fix.

**Comment by WyriHaximus** [comment]
Looks good :+1: 

Regarding driver specs and feedback I'll ask @clue and @cboden to respond with feedback or give me a list (if any).

**Comment by bwoebi** [comment]
@WyriHaximus There seems to be general approval to this PR…

So, looking forward to the feedback from @clue and @cboden, then I'll merge this if we can agree on everything (also Driver).

**Comment by cboden** [comment]
I have a busy week at work. I'll take a look as soon as I can which probably won't be until the weekend.

**Comment by bwoebi** [comment]
A quick reminder @cboden :-)

**Comment by kelunik** [comment]
Closing, as project discontinued for now.

**Pull request closed without merging**